mbuild/compound.py

```diff
@@ -146,8 +150,14 @@ class Compound(object):
         compound is the root of the containment hierarchy.
     referrers : set
         Other compounds that reference this part with labels.
+    rigid_id : int, default=None
+        Only not None if the Compound resides at the bottom of the containment 
+        hierarchy. In this case, rigid_id will refer to the integer ID of the 
+        rigid body the Compound is associated with.
```

Comment:
I think this could be phrased a bit clearer. Maybe:

```
The ID of the rigid body that this Compound belongs to. Only Particles (the bottom of the hierarchy) can have integer values for `rigid_id` set. Compounds containing rigid particles will always have `rigid_id == None`. See also `contains_rigid`.
```